bin/download.sunxi-fel

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+# The sunxi-fel tool supports passing a bus:dev to use.  However this value
```

Comment:
It is possible to grovel in sysfs and convert a "USB bus path" (ctrl-port.port.port) into the current USB device ID. I _was_ going to write such a conversion script and use it various places in this repo, but ended up adding direct support into the tools I was using for USB bus path directly and/or using udev scripts to create a constantly-named /dev symlink instead. For reference, see:

https://github.com/NVIDIA/tegrarcm/commit/cfb923c10000c5ff9925c65706992aae56422bfe
https://github.com/swarren/uboot-test-hooks/blob/master/udev/swarren-lx1/99-ubtest.rules

Anyway, this is all fine for now; the above is an FYI.